spread.yaml

```diff
@@ -57,6 +57,24 @@ backends:
                 username: ubuntu
                 password: ubuntu
 
+    external-ubuntu-core:
+        type: adhoc
+        environment:
+            SPREAD_ADHOC_UC_ADDRESS: "$(HOST: echo ${SPREAD_ADHOC_UC_ADDRESS:-localhost:8022})"
+        allocate: |
+            ADDRESS $SPREAD_ADHOC_UC_ADDRESS
+        systems:
+            # FIXME: remove the ubuntu-core-16.04-64 dummy system once the runner handles tasks without systems assigned
+            - ubuntu-core-16.04-64:
```

Comment:
Yes, we need changes in spread, currently if you have a backend with specific systems and there's one task that excludes all of them, then you get an error, for instance, if we remove the dummy system above and try to execute the suite you get:

```
fgimenez@arkham:~/Desktop/snappy_validate_image$ spread -v -reuse external:ubuntu-core-16-device-amd64
2016/09/27 09:57:44 Found /home/fgimenez/Desktop/snappy_validate_image/spread.yaml.
error: no systems specified for tests/main/interfaces-locale-control
```
The erroring task is different with each execution, probably related to the random ordering of tasks. Adding the dummy systems allow to execute the suite, `tests/main/interfaces-locale-control` is not executed anyway but doesn't throw any error.